Makefile

```diff
@@ -121,6 +122,10 @@ update-mockgen:
 	@printf $(COLOR) "Install/update mockgen tool..."
 	@go install github.com/golang/mock/mockgen@v1.7.0-rc.1
 
+update-gotestsum:
+	@printf $(COLOR) "Install/update gotestsum..."
+	@go install gotest.tools/gotestsum@latest
```

Comment:
maybe do this like the other tools:
```suggestion
@go install -modfile build/go.mod gotest.tools/gotestsum
```
so we can control the version?

or a fixed version inline here? `@latest` makes me nervous even though it's only for CI

Reply:
Good idea.